### DISCUSSION TITLE
When I run "npm run dev" nothing happens with NextJS

### DISCUSSION THREAD

**Discussion by devpedro-dias** [community/community#133987] [npm] [answered]
I was fixing some details of my final website, when suddenly I went to run it again and it appeared that "run dev" worked but it didn't appear that it was started at localhost/3000. All of this using Git bash on VsCode.
![image](https://github.com/user-attachments/assets/bcea8b11-bc17-4d3e-b13b-f35ed85f701f)

I already tried deleting the .next folder, ran "npm run dev" again and it didn't appear again. I uninstalled and installed NodeJS, I even installed node v20.16.0 and it didn't work. I also created other projects with Next JS again and it still remained the same.

This is one of my first projects and I don't know what to do, because I've already tried several ways that I saw on Stackoverflow and some discussions on GitHub and nothing worked, I'd like to know if anyone can help me.

Project Repository: https://github.com/devpedro-dias/scx-nextjs

**Comment by metagamma** [comment, marked as answer]
You can run `npm cache clean --force`. Maybe there's some other process using the same port or even a stuck process still uses it.  Try another port.

**Reply by devpedro-dias** [reply]
Thank you so much!!

**Comment by A-Kremer** [comment]
Hey,
if changing port doesn't solve it try to delete the node_modules folder and the package-lock.json file, then run npm install to reinstall the dependencies.

also maybe you are using env vars? if so, are they set corretly?

and also try to run `next build` and see maybe there are build errors that might prevent the server from starting

**Comment by urah001** [comment]
i had to restart my PC , clearing of cache didnt work for me
